src/store/RelayRecordWriter.js

```diff
@@ -94,7 +94,7 @@ class RelayRecordWriter {
    */
   getDataID(
     storageKey: string,
-    identifyingArgValue: ?string
+    identifyingArgValue: any
```

Comment:
mixed can not be passed . flow warn me `mixed` can not return to `DataID`.
but i think i should be resolved by `return stableStringify(identifyingArgValue) `.

```
getDataID( storageKey: string, identifyingArgValue: any): ?DataID {
    if (RelayNodeInterface.isNodeRootCall(storageKey)) {
      return identifyingArgValue;
    }
    ......
}
```
but i think i should be resolved by check `string` in function?
```
getDataID( storageKey: string, identifyingArgValue: mixed): ?DataID {
    if (RelayNodeInterface.isNodeRootCall(storageKey)) {
      invariant(identifyingArgValue!=null && typeof identifyingArgValue =='string',"...");
      return stableStringify(identifyingArgValue);
    }
    ......
}
```
But im not sure if its suitable for doing this.